jwql/website/apps/jwql/forms.py

```diff
@@ -37,24 +38,25 @@ def view_function(request):
 ------------
     The user must have a configuration file named ``config.json``
     placed in the ``jwql/utils/`` directory.
-"""
 
+"""
 import glob
 import os
 
+from astropy.time import Time, TimeDelta
 from django import forms
 from django.shortcuts import redirect
 
 from jwql.utils.constants import JWST_INSTRUMENT_NAMES_SHORTHAND
+from jwql.utils.engineering_database import is_valid_mnemonic
 from jwql.utils.utils import get_config, filename_parser
 
 FILESYSTEM_DIR = os.path.join(get_config()['jwql_dir'], 'filesystem')
 
 
 class FileSearchForm(forms.Form):
-    """A form that contains a single field for users to search for a proposal
-    or fileroot in the JWQL filesystem.
-    """
+    """Single-field form to search for a proposal or fileroot."""
```

Comment:
Thanks for making these docstrings more concise!

Reply:
My PEP8 checks were complaining, otherwise I would not have touched your code.